infoschema/infoschema.go

```diff
@@ -416,3 +416,21 @@ func (is *infoSchema) RuleBundles() map[string]*placement.Bundle {
 func (is *infoSchema) MockBundles(ruleBundleMap map[string]*placement.Bundle) {
 	is.ruleBundleMap = ruleBundleMap
 }
+
+// GetBundle get the first available bundle by array of IDs, possibbly fallback to the default.
+// If the default bundle is unavailable, an empty bundle with an GroupID(ids[0]) is returned.
+func GetBundle(h InfoSchema, ids []int64) *placement.Bundle {
+	for _, id := range ids {
+		b, ok := h.BundleByName(placement.GroupID(id))
+		if ok {
+			return b.Clone()
+		}
+	}
+
+	b, ok := h.BundleByName("pd")
```

Comment:
Define a constant in the `placement` package.

Reply:
Done, if the name is OK.